test/suite/wtbound.py

```diff
@@ -28,6 +28,17 @@
 
 import wiredtiger, wttest
 
+def set_prefix_bound(test, cursor, lower_bound):
+    test.assertEqual(cursor.reset(), 0)
+    cursor.set_key(lower_bound)
+    test.assertEqual(cursor.bound("action=set,bound=lower,inclusive=true"), 0)
+    # Strings are immutable in python.
+    upper_bound = list(lower_bound)
+    upper_bound[len(upper_bound) - 1] = chr(ord(upper_bound[len(upper_bound) - 1]) + 1)
```

Comment:
What does chr and ord do here? Could you add a comment explaining as well?

Reply:
There is a comment explaining, string aren't mutable in python so we represent the string as a list, ord converts the char to an int and chr converts it back to a char, we then take the string form of the list and set it as the key.

Reply:
e79451b32